src/components/TextInput.tsx

```diff
@@ -75,9 +75,9 @@ export class CTextInput extends React.Component<Props, State> {
         <RNTextInput
           ref={forwardedRef}
           style={[
+            inputStyle ?? fontStyles.regular,
```

Comment:
If the inputStyle contains `paddingVertical`, `paddingHorizontal`, `lineHeight`, `flex` value it looks like it'll be overwritten by the later `styles.input`.  I am able to overwrite the styles with the previous code if I pass `inputStyle={fontStyles.xsmall500}` or `inputStyle={{...fontStyles.xsmall500}}` to a `<TextInput ...>` such as the ones seen in `src/account/SupportContact.tsx`